multi_step_wizard/models/multi_step_wizard.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+import logging
+
+from odoo import api, fields, models
+
+_logger = logging.getLogger(__name__)
+
+
+class MultiStepWizard(models.AbstractModel):
+    """ Mixin to ease the creation of multisteps wizards
+
+    _selection_state must return all possible step of
+    the wizard.
+
+    For each state but final, there must be a method named
+    "state_exit_X" where X is the name of the state. Each
+    of these method must set the next state in self.state.
+
+    The final state has no related method because the view
+    should only display a button to close the wizard.
+
+    Look at the models and views of task.link.jira and
```

Comment:
maybe a link to the other repo?

Reply:
I actually think that example in USAGE is enough?